Sources/ProjectDescription/ResourceFileElements.swift

```diff
@@ -7,10 +7,26 @@ import Foundation
 public struct ResourceFileElements: Codable, Equatable {
     /// List of resource file elements
     public let resources: [ResourceFileElement]
+    public let excluding: [Path]
 
     public init(resources: [ResourceFileElement]) {
         self.resources = resources
+        self.excluding = []
     }
+    
+    public init(resources: [ResourceFileElement], excluding: [Path]) throws {
+        try excluding.forEach { path in
+            if path.pathString.contains("/**/") {
+                throw ResourceFileElementError.globsNotAllowed
+            }
+        }
+        self.resources = resources
+        self.excluding = []
+    }
+}
+
+public enum ResourceFileElementError: Error {
```

Comment:
This should conform to `FatalError` from `TuistSupport`

Reply:
import TuistSupport -> No such module 'TuistSupport'

Reply:
Maybe we should move the check to the `Sources/TuistLoader/Models+ManifestMappers/Target+ManifestMapper.swift`, where we actually do the logic, and where you already have the `TargetManifestMapperError`.

Reply:
Agree with @danyf90. The only role of the `ProjectDescription` models is to provide an interface for users to define their projects. Any logic like the one that checks if the format is valid should be moved to the mapping logic.